src/App/App.js

```diff
@@ -271,8 +271,14 @@ function FullApp() {
     setSelectedToPage(to);
   };
 
+  const isFirstRender = useRef(false);
```

Comment:
weird to see that isFirstRender=false for the first render and isFirstRender=true for the all other renders, it should have opposite name/value

Reply:
Sorry, I was still refactoring. It's updated now.